### PR TITLE
made edits

### PR DESCRIPTION
**Pull request by priyaayyar**
Added the missing policies to the default policies section.

### CODE REVIEW

File: modules/ROOT/pages/manage-apis-with-service-mesh.adoc [hunk 1]
```diff
@@ -30,8 +30,11 @@ image::select-policies-for-service-mesh.png[Applying Policies to APIs]
 . Select any of the following policies to secure your APIs: 
 * *Client ID Enforcement* 
 * *JWT Validation* 
-* *Rate Limit* 
-* *Rate Limit SLA-Based* 
+* *Rate Limiting* 
+* *Rate Limiting SLA-Based* 
+* *Basic Authentication: Simple*
+* *Basic Authentication: LDAP*
+* *OpenID Connect OAuth 2.0 Token Enforcement*
```
Comment (tbruno) [inline modules/ROOT/pages/manage-apis-with-service-mesh.adoc:37]: The only thing to point out is that to have `OpenID Connect OAuth 2.0 Token Enforcement` they need to have configured OpenID as their client management provider. Maybe we should add a note about that.

Comment (priyaayyar) [reply modules/ROOT/pages/manage-apis-with-service-mesh.adoc:37]: Done! Thanks!

**Review by tbruno — DISMISSED** [review]
LGTM 👍

File: modules/ROOT/pages/manage-apis-with-service-mesh.adoc [hunk 2]
```diff
@@ -30,8 +30,13 @@ image::select-policies-for-service-mesh.png[Applying Policies to APIs]
...
++
+NOTE: For the OpenID Connect OAuth 2.0 Token Enforcement policy, you must have configured OpenID as your client management provider.
```
Comment (rddheath) [inline modules/ROOT/pages/manage-apis-with-service-mesh.adoc]: When two nouns are used together as an adjective to modify a third noun in a noun string, they need a hyphen between them: client-management provider.

**Review by rddheath — DISMISSED** [review]

File: modules/ROOT/pages/manage-apis-with-service-mesh.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/manage-apis-with-service-mesh.adoc]: Done! Thanks!

**Review by rddheath — APPROVED** [review]

**Pull request merged**